.vscode/extensons.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Not sure how this got deleted, but that's not intentional. I'll take a look at this.